--- src/main/java/org/apache/maven/plugins/help/DescribeMojo.java.orig	2013-02-20 20:39:48.000000000 +0000
+++ src/main/java/org/apache/maven/plugins/help/DescribeMojo.java	2013-07-15 15:00:41.534021540 +0100
@@ -255,7 +255,7 @@
         {
             try
             {
-                project = projectBuilder.buildStandaloneSuperProject( session.getProjectBuilderConfiguration() );
+                project = projectBuilder.buildStandaloneSuperProject( localRepository );
             }
             catch ( ProjectBuildingException e )
             {
--- src/test/java/org/apache/maven/plugins/help/HelpUtilTest.java.orig	2013-02-20 20:39:48.000000000 +0000
+++ src/test/java/org/apache/maven/plugins/help/HelpUtilTest.java	2013-07-15 15:19:36.905227749 +0100
@@ -51,7 +51,7 @@
 
         MavenSession session =
             new MavenSession(
-                              container,
+                              getContainer(),
                               describe.settings, // Settings settings,
                               describe.localRepository, // ArtifactRepository localRepository,
                               null, // EventDispatcher eventDispatcher,
